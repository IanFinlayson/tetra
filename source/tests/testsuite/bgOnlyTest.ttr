def main()
  background:
    x = 0
    print("Just created x and x = ", x, "\n")
    for i in [1...100]:
      print("i = ", i, "\n")
      x += i 
      print("Just added i to x and x = ", x, "\n")
