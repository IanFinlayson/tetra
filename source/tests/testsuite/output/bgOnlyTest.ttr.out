The following error was detected in your program:
Error: (line 2) syntax error, unexpected NEWLINE, expecting ':'

Execution aborted
